### ISSUE TITLE
trpl stack frame illustrations don't render correctly

### ISSUE THREAD

**Issue by brson** [rust-lang/rust#25428]
On this page http://doc.rust-lang.org/stable/book/the-stack-and-the-heap.html

I see

```
| Address | Name | Value | +---------+------+-------+ | 4 | i | 6 | | 3 | c | 1 | | 2 | b | 100 | | 1 | a | 5 | | 0 | x | 42 
```

I don't think this is intended.

**Comment by killercup** [comment]
Should be fixed by #25395.

**Comment by steveklabnik** [comment]
Yes, this should have been fixed. :cry: about stable...
